README.md

```diff
@@ -1,61 +1,53 @@
 # ansible-aci
 
-The ansible-aci project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
+## Description
 
-This collection has been tested and supports ACI 3.2+.
-Modules supporting new features introduced in ACI API in specific ACI versions might not be supported in earlier ACI releases.
+The ansible-aci project provides an Ansible collection for managing and automating your Cisco ACI environment. It consists of a set of modules and roles for performing tasks related to ACI.
 
-*Note: This collection is not compatible with versions of Ansible before v2.8.*
+See the [Cisco.Aci collection index](https://galaxy.ansible.com/ui/repo/published/cisco/aci/content/) for a full list of modules and plugins.
 
 ## Requirements
 
-Ansible v2.14 or newer
+- Ansible v2.15 or newer
+- Python v3.10 or newer
 
-## Install
+*Note: This collection is not compatible with versions of Ansible before v2.8.*
```

Comment:
I think the requirement above of v2.15 and newer is probably overriding this note. Should we remove it?

Reply:
True, I thought it was a bit unnecessary.

Reply:
Removed